common/common.scss

```diff
@@ -481,6 +296,221 @@ hr {
   margin: 0;
   padding-bottom: 1rem;
   padding-top: 0.5rem;
+
+  .row::before {
+    display: none;
+  }
+}
+
+.p-navigation .p-navigation__link,
+.p-navigation .p-navigation__link:hover,
+.p-navigation .p-navigation__link:visited,
+.p-navigation .p-navigation__link:focus,
+.p-navigation .p-navigation__link--search-toggle,
+.p-navigation .p-navigation__link--search-toggle:hover,
+.p-navigation .p-navigation__link--search-toggle:visited,
+.p-navigation .p-navigation__link--search-toggle:focus,
+.p-navigation .p-navigation__toggle--close,
+.p-navigation .p-navigation__toggle--close:hover,
+.p-navigation .p-navigation__toggle--close:visited,
+.p-navigation .p-navigation__toggle--close:focus,
+.p-navigation .p-navigation__toggle--open,
+.p-navigation .p-navigation__toggle--open:hover,
+.p-navigation .p-navigation__toggle--open:visited,
+.p-navigation .p-navigation__toggle--open:focus,
+.p-navigation .p-navigation__dropdown-item,
+.p-navigation .p-navigation__dropdown-item:hover,
+.p-navigation .p-navigation__dropdown-item:visited,
+.p-navigation .p-navigation__dropdown-item:focus,
+.p-navigation--sliding .p-navigation__link,
+.p-navigation--sliding .p-navigation__link:hover,
+.p-navigation--sliding .p-navigation__link:visited,
+.p-navigation--sliding .p-navigation__link:focus,
+.p-navigation--sliding .p-navigation__link--search-toggle,
+.p-navigation--sliding .p-navigation__link--search-toggle:hover,
+.p-navigation--sliding .p-navigation__link--search-toggle:visited,
+.p-navigation--sliding .p-navigation__link--search-toggle:focus,
+.p-navigation--sliding .p-navigation__toggle--close,
+.p-navigation--sliding .p-navigation__toggle--close:hover,
+.p-navigation--sliding .p-navigation__toggle--close:visited,
+.p-navigation--sliding .p-navigation__toggle--close:focus,
+.p-navigation--sliding .p-navigation__toggle--open,
+.p-navigation--sliding .p-navigation__toggle--open:hover,
+.p-navigation--sliding .p-navigation__toggle--open:visited,
+.p-navigation--sliding .p-navigation__toggle--open:focus,
+.p-navigation--sliding .p-navigation__dropdown-item,
+.p-navigation--sliding .p-navigation__dropdown-item:hover,
+.p-navigation--sliding .p-navigation__dropdown-item:visited,
+.p-navigation--sliding .p-navigation__dropdown-item:focus {
```

Comment:
We're using sass which means this could be simplified by nesting selectors, e.g.:

```scss
.p-navigation--sliding {
 .p-navigation__toggle--close,
 .p-navigation__toggle--open {
   &, &:hover, &:focus {
 // ...
 }
 }
 }
 ```

But most importantly, could you explain why is this necessary?

Reply:
As demonstrated in the vanilla framework docs, this style should be included in the framework already.

https://vanillaframework.io/docs/patterns/navigation

![Google Chrome screenshot 001065@2x](https://github.com/canonical/maas-discourse-theme/assets/7452681/0b9fab8b-f7fe-4f02-b24a-076421ecb710)

Reply:
This is just to make sure that all links in every state has the black color, e.g: the `Menu` and `Close Menu` mobile navigation buttons.

Reply:
I agree though that thic can be nested, I'll work on this

Reply:
My thoughts also, it should have been included, but on inspecting and comparing with the global nav on maas.io, I realized that that wasn't the case.

Reply:
<img width="435" alt="image" src="https://github.com/canonical/maas-discourse-theme/assets/47540149/585f0c06-3380-4758-85f9-008532c58daa">
This is the menu button without the style